### ISSUE TITLE
Document how to add php support in yeoman

### ISSUE THREAD

**Issue by idflood** [fgnass/gateway#1]
Hi,

I followed http://fgnass.github.com/2012/08/30/yeoman.html but I think some documentation is missing. 

How did you modify yeoman so that it handle php files? I believe yeoman/tasks/server.js needs to be modified but how?

**Comment by fgnass** [comment]
https://github.com/fgnass/yeoman/commit/39faa07375ee49d56bd50d334dcfd6b5a4e96292

**Comment by idflood** [comment]
Awesome, thanks!

**Comment by idflood** [comment]
One more question. It seems that $_POST values are never set. I tried to create a basic form with "get" method and it worked but if I change it to "post" the the variables are not set. (I tested it by doing a <?php print_r(get_defined_vars()); ?>)

Have you ever encountered this behavior?

**Comment by idflood** [comment]
So I made different tests but the $_POST variables are still not defined.
- Doing a phpinfo() shows that the right php is running with normal settings (post_max_size at 32M, max_input_vars at 1000, ...)
- Doing a console.log(req); in gateway.js just after defining the "env" variable doesn't show any _POST variable. ( I tried with other available variables but with the same result )

**Comment by idflood** [comment]
I have found some interesting things:
- There is a connect middleware "bodyParser": http://www.senchalabs.org/connect/middleware-bodyParser.html
- Adding it in yeoman before the gateway(opts.base) gives some result
- After that, in gateway.js "req.body" contains the $_POST information.

Now I tried to add it in the env variable like the others but without success ( POST: req.body ). I believe there is not much left to have working POST data, but when it will be good I will have to test file upload too.

```
middleware = middleware.concat([
  connect.bodyParser(),
  // Serve PHP files via php-cgi.
  gateway(opts.base),
```
